tests/utils.ts

```diff
@@ -8,3 +8,13 @@ export function isSshDirFound () {
         return false;
     }
 }
+
+
+export function stripAnsi (str: string) {
+    // Copied from https://github.com/chalk/ansi-regex/blob/main/index.js#L1-L8
```

Comment:
I can't figure out how to import the package https://github.com/chalk/strip-ansi ... ><

Reply:
ESM modules ftw :smile: 

Hope nodejs 22 manages to make commonjs and esm interoperable

Reply:
You can try https://tsx.is/ which allows you to have interop. This way you also don't need `tsc` step to run `ts` files
```
node --import tsx ...
```
https://github.com/nodejs/node/blob/main/doc/api/typescript.md

Reply:
This am not that familiar with this and not too sure what you mean. 

But this is the exact problem that i am facing
https://github.com/firecow/gitlab-ci-local/commit/6d7e6a7412bae17682c5c210567bb326822dcedc

![image](https://github.com/user-attachments/assets/fb44cfb3-b45b-45a3-9268-c2f9238ab2ac)